core/operation/src/main/java/uk/gov/gchq/gaffer/jobtracker/JobTracker.java

```diff
@@ -55,7 +56,7 @@ public JobTracker(final String cacheNameSuffix) {
     public void addOrUpdateJob(final JobDetail jobDetail, final User user) {
         validateJobDetail(jobDetail);
         try {
-            CacheServiceLoader.getService().putInCache(cacheName, jobDetail.getJobId(), jobDetail);
+            super.addToCache(jobDetail.getJobId(), jobDetail, true);
```

Comment:
Why is the CacheServiceLoader not used anymore? Is this because the Cache class uses it anyway?

Reply:
There was duplicate implementations of the cache.